src/coreclr/src/dlls/mscoree/coreclr/CMakeLists.txt

```diff
@@ -164,6 +164,29 @@ endif(FEATURE_EVENT_TRACE)
 
 target_link_libraries(coreclr ${CORECLR_LIBRARIES})
 
+# Create the runtime module index header file containing the coreclr build id
+# for xplat and the timestamp/size on Windows.
+if(CLR_CMAKE_HOST_LINUX OR CLR_CMAKE_HOST_WIN32)
```

Comment:
Do we have a `CLR_CMAKE_HOST_ELF_PLATFORM` (or something), or do we really rely on Linux-only syscalls to make this thing work?

Reply:
No there isn't. This is a kind of a special condition of both Windows and Linux (but not MacOS and the FreeBSD). It wouldn't be worth it to add a special flag.